modules/private-access/pages/gcp.adoc

```diff
@@ -21,29 +21,63 @@ In the GCP portal, create a Private Service Connect endpoint using the service n
+Google has a detailed page about using Private Service Connect at this link: link:https://cloud.google.com/vpc/docs/configure-private-service-connect-services#create-endpoint[Create a Private Service Connect Endpoint]
 
-In the GCP portal, search for btn:[Private Service Connect] and click btn:[Connect Endpoint].
+. Verify that a subnet with *Purpose* “None“ exists to contain the Private Service Connect endpoint.
+.. The subnetwork region must be the same as TigerGraph’s cloud. You can reuse an existing subnetwork.
 
-In the btn:[Target] field, choose btn:[Published service]. Use the service name provided by TigerGraph as the btn:[Target Service].
-Choose the Subnetwork and IP address services provided earlier. Click btn:[Add Endpoint].
+. In the Google Cloud web interface, go to *Private Service Connect*.
 
-Return to the Private service connect section and allow a few minutes to wait. On approval, the Endpoint displays a Status value of Accepted.
+. Go to the *Connected endpoints* tab and click btn:[Connect endpoint].
 
-==== Create endpoint via CLI
+. For *Target*, select *Published service*.
 
-First create a subnetwork:
+. For *Target service*, enter the service attachment URI from the TigerGraph Cloud Portal.
+.. The service attachment URI is in this format: `projects/SERVICE_PROJECT/regions/REGION/serviceAttachments/SERVICE_NAME`
+
+. Enter a name to use for the endpoint under *Endpoint name*
+
+. Select a *Network* for the endpoint. This is the endpoint that connects to the TigerGraph cluster.
+
+. Select a Subnetwork for the endpoint from step 1.
+
+. Select an IP address for the endpoint. If you need a new IP address, you can create one:
+
+.. Select btn:[Create IP address] from the *IP address* drop-down menu
+
+.. Enter a *Name* and optional *Description* for the IP address.
+
+.. For Static IP address, select either btn:[Assign automatically] or enter your own with btn:[Let me choose].
+
+.. Click btn:[Reserve] to finish the IP address setup.
+
+. Click btn:[Add endpoint] to complete the process.
+
+=== Private DNS via GCP console
+Create a Private DNS Zone named `privatelink.i.tgcloud.io` on the *Cloud DNS* page.
+
+In the new DNS zone, add a DNS *A* record with the same name as the solution domain from the TigerGraph Cloud Portal.
+The value is the private static IP address for Private Service Connect.
+
+[WARNING]
+Do not add `privateline.tgcloud.io` to the DNS *A* record.
+
+
+=== Create a connection via CLI
+
+Verify that a subnet with *Purpose* set to *None* exists to contain the Private Service Connect endpoint. The subnet region must be the same as TigerGraph Cloud.
+
+You can reuse an existing subnet that you have already created using the GUI or create a new subnetwork using this command:
 
 [source.wrap, bash]
 ----
 gcloud compute networks subnets create <subnetwork_name> --project=<customer_project> --range=<subnet_cidr_in_customer_network> --network=<customer_network> --region=<region>
```

Comment:
For CLI based solutions, should maybe mention that they need to have installed the CLI tools first?Maybe provide a link on how to install cli tools like gcloud